### DISCUSSION TITLE
ModuleNotFoundError: No module named 'milvus_lite'

### DISCUSSION THREAD

**Discussion by jasonwjli119** [milvus-io/milvus#36759] [Q&A and General discussion] [answered]
I run below code follow the instruction and got error 
Failed to create new connection using: ae99195a1b35496587ccdf192c5d5ad1
ModuleNotFoundError: No module named 'milvus_lite', any idea?

pip install -U pymilvus

from pymilvus import MilvusClient
client = MilvusClient("./milvus_demo.db")

**Comment by yhmo** [comment, marked as answer]
Your OS is Windows?
The milvus-lite [readme](https://github.com/milvus-io/milvus-lite/blob/main/README.md) says it only supports Ubuntu and MacOS. So, in windows os, the milvus-lite package won't be installed along with pymilvus.
```
Milvus Lite currently supports the following environments:
- Ubuntu >= 20.04 (x86_64 and arm64)
- MacOS >= 11.0 (Apple Silicon M1/M2 and x86_64)
```

**Reply by jasonwjli119** [reply]
Yes, my OS is Windows. Thanks, yhmo.

**Comment by Telanx** [comment]
so it cannot run on windows?

**Reply by yhmo** [reply]
This is milvus-lite repo: https://github.com/milvus-io/milvus-lite
In the README, it mentioned: 
```
Milvus Lite currently supports the following environmnets:

Ubuntu >= 20.04 (x86_64 and arm64)
MacOS >= 11.0 (Apple Silicon M1/M2 and x86_64)
Note: Windows is not yet supported.
```

**Comment by AyushAP8** [comment]
how can I run milvus-lite on windows then ?

**Reply by yhmo** [reply]
So far, milvus-lite doesn't support Windows.
https://github.com/milvus-io/milvus-lite?tab=readme-ov-file#requirements

**Comment by tobiaswuerth** [comment]
would love to see support for windows <3

**Comment by andyzhu** [comment]
i follow the same instruction on MacOS, it has the same problem. it requires to install milvus-lite seperately

**Reply by yhmo** [reply]
Seems the milvus-lite has been moved out from pymilvus recently by this commit: https://github.com/milvus-io/pymilvus/pull/2976
You can install it manually:  `pip install -U milvus-lite`
